components/suggest/src/schema.rs

```diff
@@ -28,6 +28,17 @@ CREATE TABLE meta(
     value NOT NULL
 ) WITHOUT ROWID;
 
+CREATE TABLE rs_cache(
+    collection TEXT PRIMARY KEY,
+    data TEXT NOT NULL
+) WITHOUT ROWID;
+
+CREATE TABLE ingested_records(
```

Comment:
Should we add a field for `collection` here? I am a little concerned that things will mess up if we pull `Fakespot` out of the enums of `SuggestRecord` and `SuggestRecordType` in the future. If you disagree, please add a comment here to note that.